docs/src/config/sidebar.yml

```diff
@@ -2,32 +2,36 @@
 
 - label: "Introduction"
   link: "/"
-- label: Usage
+- label: Welcome
   items:
     - label: "Install"
       link: "/usage/install/"
-    - label: "Quick Start"
-      link: "/usage/quick-start/"
     - label: "Integrations"
       link: "/usage/integrations/"
-    - label: "Linters"
-      link: "/usage/linters/"
-    - label: "Configuration"
-      link: "/usage/configuration/"
-    - label: "False Positives"
-      link: "/usage/false-positives/"
-    - label: "Performance"
-      link: "/usage/performance/"
+    - label: "Quick Start"
```

Comment:
Maybe we should position "Quick Start" before "Integrations". As a newcomer, I would like to try the tool first, and then integrate it into my IDE.